docs/pbs/preemption.md

```diff
@@ -28,6 +28,12 @@ however, in that:
   resources being used are required by a higher priority job in a
   different queue.
 
+- Specifically, when the running job is targeted for
+  preemption, the scheduler will send a termination signal (`SIGTERM`),
+  wait 10 minutes, and then send a kill signal (`SIGKILL`), finally ending the job.
+    - The user has the option to detect and act upon the `SIGKILL` using the methods
```

Comment:
I assume this should read SIGTERM? A SIGKILL is not actionable.

Reply:
doh, right!!